web/docs/guides/examples.mdx

```diff
@@ -35,6 +35,7 @@ Build a basic Todo List with Supabase and your favorite frontend framework:
 - Kotlin (in development): [GitHub](https://github.com/supabase/supabase-kt)
 - `useSupabase`: Supabase React Hooks. [GitHub](https://github.com/gbibeaul/use-supabase)
 - `vue-supabase`: Supabase Vue wrapper. [GitHub](https://github.com/supabase/vue-supabase)
+- `vue-3-supabase`: Supabase Vue 3 wrapper. [GitHub](https://github.com/DidoMarchet/vue-3-supabase)
```

Comment:
Hey @DidoMarchet - this is the only part we need!

You can remove everything else in the `examples` folder (those are the examples we officially support)

Reply:
Hi!
I do it!
Thanks and have a nice day,

Davide

Reply:
thanks Davide. 

> Thanks and have a nice day,

you have a nice day too  :)